frontend/mobilityai_react/App/Lib/Api.js

```diff
@@ -1,14 +1,15 @@
 import RNFetchBlob from 'rn-fetch-blob';
 
-const SERVER_URL = "https://localhost:5001/";
+// const SERVER_URL = "https://localhost:5001/";
+const SERVER_URL = "https://mobilityai.teovoinea.com/";
```

Comment:
We should use dev/prod env vars to change the url accordingly

Reply:
Not sure how to do this, but I can take a look at it later.